packages/contracts-bedrock/src/L1/LiquidityMigrator.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ISharedLockbox } from "./interfaces/ISharedLockbox.sol";
+
+/// @custom:proxied true
```

Comment:
I dont know if this is needed here, because it is not a permanent contract, but it will be used with a proxy this time tho